packages/table/src/tableState.ts

```diff
@@ -94,6 +94,8 @@ export interface TableState {
     columnIdToIndex: { [key: string]: number };
 
     childrenArray: Array<React.ReactElement<ColumnProps>>;
+
+    scrollDirection?: "TOP" | "BOTTOM" | "LEFT" | "RIGHT" | "NONE";
```

Comment:
let's save this type as a const enum somewhere in the table package so that it's not duplicated in multiple places:

```ts
export enum ScrollDirection {
  TOP = "top",
  BOTTOM = "bottom",
  // ...
}
```

Reply:
makes sense!